README.es.md

```
@@ -157,21 +157,21 @@ career-ops es un unico slash command con multiples modos:
 /career-ops deep           → Research profundo de empresa
 ```
 
-O simplemente pega una URL o descripcion de oferta -- career-ops la detecta y ejecuta el pipeline completo.
+O simplemente pega una URL o descripción de oferta -- career-ops la detecta y ejecuta el pipeline completo.
 
-## Como funciona
+## Cómo funciona
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)